DQM/TrackingMonitor/python/TrackFoldedOccupancyClient_cfi.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+
+TrackerMapFoldedClient = DQMEDHarvester("TrackFoldedOccupancyClient",  
+    FolderName = cms.string('Tracking/TrackParameters'),
+    AlgoName = cms.string('GenTk'),
+    TrackQuality = cms.string('generalTracks'),
+    PhiMax = cms.double(3.141592654),
+    PhiMin = cms.double(-3.141592654),
+    EtaMax = cms.double(2.5),
```

Comment:
are you sure about this ? should not we want 3.0 ? (at least for Phase1) and possibly 4.0 for Phase2